### ISSUE TITLE
Toast button text can not be empty string

### ISSUE THREAD

**Issue by MilkBiscuit** [GeekyAnts/NativeBase#1351]
Hi! Thanks for trying out NativeBase!
Due to the overwhelming requests and issues/questions we ask you to add the following details with your issue.

Please check the existing open/closed issues for a possible duplicate before creating a new issue :)

Also **DO** fill out the below form to give us a better idea about your environment and help us debug it quicker. Issues without the required details will mostly be unattended to.

## react-native, react and native-base version
react-native-cli: 2.0.1
react-native: 0.46.4

## Expected behaviour
button text is an optional value, or I could set empty string to it.

## Actual behaviour
Runtime exception

## Steps to reproduce (code snippet or screenshot)
I wrote a function to show toast to user, if I write code like below:
```
export const showToast = (text, buttonText = '', duration = 1500) => {
  Toast.show({
    text,
    position: 'bottom',
    buttonText,
    duration
  });
};
```
I got runtime error like this:
![simulator screen shot - iphone 6 - 2017-11-08 at 21 10 55](https://user-images.githubusercontent.com/20746964/32538083-b3de6374-c4c9-11e7-9359-57e96adf00f8.png)

If change code to below, everything works fine
```
export const showToast = (text, buttonText = 'Okay', duration = 1500) => {
  Toast.show({
    text,
    position: 'bottom',
    buttonText,
    duration
  });
};
```

## Is the bug present in both ios and android or in any one of them?
on IOS, did not check android

**Comment by akhil-geekyants** [comment]
@MilkBiscuit If you don't wan't to show button text, you can simply pass a string with a space as button text.

![toast](https://user-images.githubusercontent.com/27768184/32727213-c15326b2-c8a1-11e7-9898-649d237efe2e.gif)

**Comment by SupriyaKalghatgi** [comment]
Fixed with NativeBase [2.3.5](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.5)
